src/App.jsx

```diff
@@ -1,35 +1,44 @@
-import * as React from 'react';
-import TodoList from './TodoList';
-import AddTodoForm from './AddTodoForm';
+import { useState, useEffect } from 'react'
+import reactLogo from './assets/react.svg'
+import viteLogo from '/vite.svg'
```

Comment:
Remove these imports, they're not being used anymore.

Reply:
Done
Thank you Gabriel!